secure/unprivileged-building-of-container-images-using-buildah.adoc

```diff
@@ -6,15 +6,27 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Running {pipelines-shortname} as the root user on a container can expose the container processes and the host to other potentially malicious resources. You can reduce this type of exposure by running the workload as a specific non-root user in the container. To run builds of container images using Buildah as a non-root user, you can perform the following steps:
+Running {pipelines-shortname} as the root user on a container can expose the container processes and the host to other potentially malicious resources. You can reduce this type of exposure by running the workload as a specific non-root user in the container.
+
+In most cases, creating a custom task for building the image and configuring user namespaces in this task enables running Buildah without root privileges.
+
+If your image does not build successfully using this configuration, you can define custom service account (SA) and security context constraint (SCC); however, in this option, you must enable the Buildah step to raise its privileges (`allowPrivilegeEscalation: true`).
```

Comment:
i'm not sure if the options you're referring to is the latter SCC or both of them. my suggestion is to make that explicit (i guessed it's for both of them, but i might be wrong 😄)
```suggestion
If your image does not build successfully using this configuration, you can define a custom service account (SA) and a security context constraint (SCC). However, in either of these cases, you must enable the Buildah step to raise its privileges (`allowPrivilegeEscalation: true`).
```

Reply:
This is a single option. I did some slight rewording to make this fact clearer.